tutorials/roofit/rf710_roopoly.C

```diff
@@ -0,0 +1,67 @@
+/// \file
+/// \ingroup tutorial_roofit
+/// \notebook -js
+/// Taylor expansion of RooFit functions using the taylorExpand function with RooPolyFunc
+///
+/// \macro_image
+/// \macro_output
+/// \macro_code
+///
+/// \date December 2021
+/// \author Rahul Balasubramanian
+
+#include "RooRealVar.h"
+#include "RooPolyFunc.h"
+#include "RooAbsCollection.h"
+#include "RooPlot.h"
+#include "TCanvas.h"
+
+using namespace RooFit;
+
+void rf710_roopoly()
+{
+   // C r e a t e   p o l y n o m i a l
+   // f u n c t i o n  o f  f o u r t h  o r d e r
+   // ---------------------------------------------
+   // x^4 - 5x^3 + 5x^2 + 5x -6
+   // ---------------------------------------------
+
+   RooRealVar x("x", "x", 0, -3, 10);
+   RooPolyFunc f("f", "f", RooArgSet(x));
+   f.addTerm(+1, x, 4);
+   f.addTerm(-5, x, 3);
+   f.addTerm(+5, x, 2);
+   f.addTerm(+5, x, 1);
+   f.addTerm(-6, x, 0);
+
+   // C r e a t e   t a y l o r  e x p a n s i o n
+   // ---------------------------------------------
+   double x0 = 2.0;
+   auto taylor_o1 = RooPolyFunc::taylorExpand(f, 1, RooArgSet(x), x0);
+   auto taylor_o2 = RooPolyFunc::taylorExpand(f, 2, RooArgSet(x), x0);
```

Comment:
```suggestion
   auto taylor_o1 = RooPolyFunc::taylorExpand(f, 1, RooArgSet(x), x0);
   auto taylor_o2 = RooPolyFunc::taylorExpand(f, 2, RooArgSet(x), x0);
```
Why do you only go up to the 2nd order here? I think it would be nicer to go up to the 4th order (as suggested in https://github.com/root-project/root/pull/9184#issuecomment-951757561), so you can show how your Taylor expansion matches the original polynomial.

Reply:
Currently only first and second order expansion is supported. This shall be documented :)